app/views/rating_questions/index.html.erb

```diff
@@ -1,7 +1,12 @@
 <div class="flash-notice"><%= flash[:notice] %></div>
```

Comment:
Move these flash notice / alert divs into the application layout: `app/views/layouts/application.html.erb`. That way then, they will be rendered on all views, not just this one.